qt4/backintime-qt4_polkit

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ $(which pkexec) ]; then
+        pkexec --disable-internal-agent "/usr/bin/backintime-qt4" "$@"
```

Comment:
Got it, thanks.

It's worth noting that while ```gparted``` is called with ```--disable-internal-agent``` on your system, on my system (and presumably other Ubuntu systems), ```gparted``` is NOT called with ```--disable-internal-agent```. I am certainly not a ```pkexec``` expert, but I'd be interested in a quick survey of how ```pkexec``` is used to activate authentication agents among different distributions before changing the way that we use it for everyone.

What distro are you using?